### PR TITLE
Fixes for PUI CI run/login timeouts

### PR DESCRIPTION
**Pull request by matmair**
This fixes recent CI problems and login timeouts related to recent changes in that are. See https://github.com/inventree/InvenTree/pull/5749#issuecomment-1778328406 or any other recent PUI CI run

### CODE REVIEW

**Comment by matmair** [comment]
@SchrodingersGat this fixes teh PUI CI issues - the problem is a mixture of long API timeouts and reloads caused by that.

File: src/frontend/src/main.tsx [hunk 1]
```diff
@@ -25,26 +25,37 @@ export const IS_DEV = import.meta.env.DEV;
...
-          demo: {
+          'mantine-u56l5jt85': {
             host: 'https://demo.inventree.org/',
             name: 'InvenTree Demo'
           }
         }
       : {})
   },
-  default_server: IS_DEMO ? 'demo' : 'localhost',
+  default_server: IS_DEMO ? 'mantine-u56l5jt85' : 'mantine-cqj63coxn',
```
Comment (SchrodingersGat) [inline src/frontend/src/main.tsx:54]: Why the pseudo-random map keys here?

Comment (matmair) [reply src/frontend/src/main.tsx:54]: Way less likely to be used by the server_list provided by the backend. Was a problem for me already when debugging this on my Q enviroment.

**Comment by SchrodingersGat** [comment]
Thanks for the fix @matmair

**Pull request merged**